Packages.props

```diff
@@ -13,6 +13,7 @@
     <PackageReference Update="Shouldly" Version="4.0.3" />
     <PackageReference Update="StyleCop.Analyzers" Version="1.1.118" />
     <PackageReference Update="System.IdentityModel.Tokens.Jwt" Version="6.6.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.OAuth" Version="2.2.0" />
```

Comment:
This is redundant, the providers target ASP.NET Core 5 where this is built in.
```suggestion
```

Reply:
Removed